charts/rancher-istio/v1.1.1/charts/kiali/templates/deployment.yaml

```diff
@@ -39,6 +39,10 @@ spec:
         - "-v"
         - "4"
         env:
+        - name: JAEGER_URL
+          value: 'https://{{ .Values.global.rancher.domain }}/k8s/clusters/{{ .Values.global.rancher.clusterId }}/api/v1/namespaces/{{ .Release.Namespace }}/services/tracing:80/proxy/jaeger'
+        - name: GRAFANA_URL
+          value: 'https://{{ .Values.global.rancher.domain }}/k8s/clusters/{{ .Values.global.rancher.clusterId }}/api/v1/namespaces/{{ .Release.Namespace }}/services/grafana:80/proxy'
```

Comment:
Have you test this? We used to use grafana service dns url, it worked. We need to check how Kiali use these URL.